index.html

```diff
@@ -6,74 +6,231 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Summer Camp</title>
-  <link rel="apple-touch-icon" sizes="180x180" href="./Media/favicon_io/apple-touch-icon.png">
-  <link rel="icon" type="image/png" sizes="32x32" href="./Media/favicon_io/favicon-32x32.png">
-  <link rel="icon" type="image/png" sizes="16x16" href="./Media/favicon_io/favicon-16x16.png">
+  <link rel="apple-touch-icon" sizes="180x180" href="./favicon_io/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon_io/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon_io/favicon-16x16.png">
   <link rel="stylesheet" href="style.css">
   <script src="https://kit.fontawesome.com/b89b7c8adf.js" crossorigin="anonymous"></script>
   <link xmlns="http://www.w3.org/1999/xhtml" rel="stylesheet" href="styles.css" />
   <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+  <meta name="googlebot" content="notranslate" />
 </head>
 
 <body>
+
+  <!--Pre Navigation Bar-->
+
   <div class="pre-nav">
-    <a href=""> <i class="fab fa-twitter"></i></a>
-    <a href=""><i class="fab fa-instagram"></i></a>
+    <a title="Instagram" href="https://www.instagram.com/adarsh8090adi/" target="_blank" rel="noopener noreferrer"><i
+        class="fab fa-instagram"></i></a>
+    <a title="Facebook" href="https://www.facebook.com/people/%C3%82d%C3%A0rsh-P%C3%A1th%C3%A3k/100010279544328/"
+      target="_blank" rel="noopener noreferrer"><i class="fa-brands fa-facebook"></i></i></a>
+    <a title="Linkedin" href="https://www.linkedin.com/in/adarsh-pathak-56a831256/"><i class="fa-brands fa-linkedin"
+        target="_blank" rel="noopener noreferrer"></i></i></a>
+    <a title="Twitter" href="https://twitter.com/PowerLevel9002?t=AIuSN7mTxk5a_MWpLolEjA&s=09" target="_blank"
+      rel="noopener noreferrer"><i class="fab fa-twitter"></i></a>
     <ul class="pre-menu">
-      <li id="google_translate_element"></li>
-      <li><a href="#">My Page</a></li>
-      <li><a href="#">Input</a></li>
+      <li title="Google Translator" id="google_translate_element"></li>
+      <li title="My Portfolio"><a href="https://powerlevel9000.github.io/portfolio-project" target="_blank"
+          rel="noopener noreferrer">My Page</a></li>
+      <li title="Registration Page"><a href="#">Input</a></li>
     </ul>
   </div>
```

Comment:
# I consider those changes 

## My POV
Adjusting both alignments looks weird to me as my pre-nav contains lots of things so I specifically try to make it in the corner so that it is not part of the web page because it contains information about me not about the event also on taking a close look I find out that my social icons should have the same margin 😁😁